### PR TITLE
Dm 2999

### PR DESCRIPTION
**Pull request by JoshingYou1**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-2999

## Description - what does this code do?
Moves the diffusion map filters into an accordion and pushes the map down on the screen.

## Testing done - how did you test it/steps on how can another person can test it 
1. Visit `/diffusion-map`.
2. Make sure the practice list is in its own scrollable container.
3. Make sure when you reset/update the filters, the accordion gets closed.
4. Make sure the filters still work as intended.

## Acceptance criteria
- Have accordion push map down as opposed to hovering over map like how the Search is
- Use USWDS accordion filter
- Redesign layout of buttons
- Put Innovations in their own scrollable container
- Have design/UX review
- Implement 

## Definition of done
- [X] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [X] A link has been provided to the originating JIRA issue
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by bradjohnson92008 — APPROVED** [review]
Looks great Josh.. nice work!

File: app/assets/stylesheets/dm/pages/_diffusion_map.scss [hunk 1]
```diff
@@ -17,3 +17,15 @@
...
   }
 }
+
+.map-filters-accordion {
+  h4 {
+    @include u-maxw('full', !important);
+  }
+
+  .usa-accordion__content {
+    border-bottom: 0.25rem solid #f0f0f0 !important;
```
Comment (emyl3) [inline app/assets/stylesheets/dm/pages/_diffusion_map.scss:27]: i believe `#f0f0f0` is `$theme-color-base-lightest`

File: app/views/maps/diffusion_map.html.erb [hunk 2]
```diff
@@ -17,15 +17,44 @@
...
+          (of <%= @dh_markers.size %>)
+        </p>
+        <p class="margin-0">
+        <span id="practice-results-count">
+          <%= diffused_practices.size %> innovation<%= 's' unless diffused_practices.size === 1 %>
+        </span>
+          matched (of <%= diffused_practices.size %>)
+        </p>
+      </div>
+<!--      <button id="filterResultsTrigger" class="map-filters-trigger usa-button width-full display-block margin-bottom-105">-->
```
Comment (emyl3) [inline app/views/maps/diffusion_map.html.erb:50]: nit: we can remove this if we aren't using it

**Review by emyl3 — CHANGES_REQUESTED** [review]
a couple of nits! Great work josh!

File: app/assets/stylesheets/dm/pages/_diffusion_map.scss (hunk 1 above)
Comment (JoshingYou1) [reply app/assets/stylesheets/dm/pages/_diffusion_map.scss]: Okay, I changed it 😄

File: app/views/maps/diffusion_map.html.erb (hunk 2 above)
Comment (JoshingYou1) [reply app/views/maps/diffusion_map.html.erb]: Removed. Thank you! 😸

**Review by emyl3 — APPROVED** [review]
Nice work! Thanks, Josh!

**Pull request merged**